mail/gophish/files/gophish.init

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2006-2011 OpenWrt.org
+
+START=50
+STOP=50
+
+USE_PROCD=1
+PROG=/usr/sbin/gophish
+
+start_service() {
+	user_exists gophish || user_add gophish
```

Comment:
Shouldnt you rather create this user in Makefile?

Reply:
The Makefile now uses USERID instead.